samples/SocketsSample/.gitignore

```diff
@@ -0,0 +1 @@
+wwwroot/js
```

Comment:
Can we be consistent about this? I don't like the per project git ignore. Lets just keep the global signalr-client.js include for now...